lib/cisco_node_utils/cmd_ref/vxlan_vtep.yaml

```diff
@@ -0,0 +1,26 @@
+# vxlan_vtep
+---
+all_interfaces:
+  multiple:
+  config_get: 'show running-config interface all'
+  config_get_token: '/^interface\s+(nve.*)$/'
+
+mac_distribution:
+  config_get: 'show running-config interface all'
+  config_get_token: ['/^interface <name>$/', '/^host-reachability protocol (\S+)/']
+  config_set: ['interface <name>', '<no_cmd> host-reachability protocol <proto>', 'end']
+
+#TODO: move to vxlan_vtp_vni.rb
```

Comment:
Comments need a space after the # (rubocop should catch that one). Any chance of capitalizing the beginning where possible?